libkbfs/key_manager_test.go

```diff
@@ -1077,6 +1077,92 @@ func TestKeyManagerReaderRekey(t *testing.T) {
 	}
 }
 
+func TestKeyManagerReaderRekeyAndRevoke(t *testing.T) {
```

Comment:
Why do we need to test revocation as part of this case? I would have thought we can do the following:

1. Create u1d1 and u2d1.
2. Write file to u1#u2 from u1d1.
3. Create u1d2.
4. Create u2d2.
5. Rekey from u2d2.
6. Rekey again from u2d2, check that revision hasn't changed.